the-collab-lab/tcl-17-smart-shopping-list
# Rr style welcome page

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description
This PR adds styling to the welcome page including a background image, color, and uses the Material UI components `Box`, `Button`, `Container`, and `TextField`. 


## Related Issue
closes #59 


## Acceptance Criteria
- [ ] The user sees a background image instead of a lot of white space
- [ ] The user sees the app name
- [ ] The user sees fun colors rather than just black and white
- [ ] The user sees elements (like the buttons, text inputs) that are the same throughout the app



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
## Updates

### Before
![welcomeScreenshotBefore](https://user-images.githubusercontent.com/11085221/109375143-36bd2300-786f-11eb-8d96-b5d2db90349e.JPG)




### After
![welcomeScreenshot](https://user-images.githubusercontent.com/11085221/109375145-3de43100-786f-11eb-81e8-a389c816fd9a.JPG)




## Testing Steps / QA Criteria
* One way to test the site is to click on the `details` button below to open the `Netlify preview`
![Screen Shot 2021-01-12 at 6 05 51 PM](https://user-images.githubusercontent.com/14856004/104389400-ff81f500-5500-11eb-95ae-4c0169a069cb.png)

* Otherwise from your terminal, pull down this branch with `git pull origin RR-style-welcome-page` and check that branch out with `git checkout RR-style-welcome-page`
* Run `npm start` in your terminal to launch the application in the browser and see the welcome page
